### DISCUSSION TITLE
TL-WN725N On BigSur

### DISCUSSION THREAD

**Discussion by Appentium** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#200] [General]
After many searching, finally I found you and bought TL-WN725N. Its just working like magic on my Hackintosh BigSur. Thank you

**Comment by obeders** [comment]
Boa noite, vc baixou a Kast direto do site(Fabricante) ou daqui, pq eu estou tentando com uma PCI que esta no note, e nada até agora.  O meu é AR9287 atherus.   agradeço
